pennylane/qchem/integrals.py

```diff
@@ -138,14 +138,29 @@ def _generate_params(params, args):
     Returns:
         list(array[float]): basis set parameters
     """
+    # Friendly reference for JAX.
+    # params order is ALPHA COEFF COORD
+    # args order is COORD COEFF ALPHA
```

Comment:
While going through the codebase, do you think that the opposite `argument` ordering for the `autograd` can be changed to have a uniform ordering for both interfaces?

Reply:
Yes, they can be changed to have a uniform ordering, however it is very tricky IMO since I would have to basically look at every function that calls _generate_params, of which there are a lot, and see what the assumed args are. In practice this would be a lot of work essentially refactoring all of the autograd workflow. Since at one point we're going to remove the autograd workflow, I'd rather just keep it as is.